### PR TITLE
New package: python-discid-1.1.1

### PR DESCRIPTION
**Pull request by Johnnynator**

### CODE REVIEW

**Comment by cr6git** [comment]
Is this package a dependency of `picard` ? https://github.com/void-linux/void-packages/pull/1041

**Comment by Johnnynator** [comment]
It is a optional dependency.

**Comment by Johnnynator** [comment]
Picard works fine without.

**Comment by cr6git** [comment]
Thanks!

File: srcpkgs/python-discid/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'python-discid'
+pkgname=python-discid
```
Comment (cr6git) [inline srcpkgs/python-discid/template]: But why the name is not `python3-discid` ?

File: srcpkgs/python-discid/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+version=1.1.1
+revision=1
+noarch=yes
+wrksrc="discid-${version}"
+build_style=python3-module
+pycompile_module=discid
+hostmakedepends="python3 libdiscid"
+depends="libdiscid"
+short_desc="Python binding of libdiscid by MusicBrainz"
```
Comment (cr6git) [inline srcpkgs/python-discid/template]: `"Python3 binding ..."` would be more precise.

**Comment by Johnnynator** [comment]
Added a occasional `3` here and there.

**Pull request merged**